[jit] Fixed a bug in the pass that replaces cat with the variadic op

navahgar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#61795**

Differential Revision: [D29748785](https://our.internmc.facebook.com/intern/diff/D29748785)

navahgar committed:
[jit] Fixed a bug in the pass that replaces cat with the variadic op

[ghstack-poisoned]

eellison reviewed (approved):
Darn short circuiting

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29748785).

navahgar committed:
Update on "[jit] Fixed a bug in the pass that replaces cat with the variadic op"


Differential Revision: [D29748785](https://our.internmc.facebook.com/intern/diff/D29748785)

[ghstack-poisoned]

navahgar commented:
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29748785).
